packages/svelte/tests/runtime-runes/samples/svelte-element-draggable/_config.js

```diff
@@ -0,0 +1,13 @@
+import { ok, test } from '../../test';
+import { flushSync } from 'svelte';
+
+export default test({
+	html: `<div draggable="false"></div>`,
+
+	async test({ assert, target, logs }) {
+		const div = target.querySelector('div');
+		ok(div);
+
+		assert.deepEqual(div.getAttribute('draggable'), 'false');
```

Comment:
I think we probably want to assert that the property is false:

```suggestion
		assert.deepEqual(div.draggable, false);
```

Reply:
Well the original issue was actually about the attribute but i guess the property can be the same...maybe both? Also should i add the spread to this test, it was also broken?

Reply:
Updated the test to check both the property and the attribute and i've added the spread attributes to the check.

Reply:
nice, good idea